src/Frontend/Themes/Bootstrap4/Modules/Blog/Layout/Templates/Detail.html.twig

```diff
@@ -0,0 +1,168 @@
+{#
+variables that are available:
+- {{ item }}: contains data about the post
+- {{ comments }}: contains an array with the comments for the post, each element contains data about the comment.
+- {{ commentsCount }}: contains a variable with the number of comments for this blog post.
+- {{ navigation }}: contains an array with data for previous and next post
+#}
+{% import 'Core/Layout/Templates/Alerts.html.twig' as alerts %}
+<div id="blogDetail" class="blog blog-detail">
+  <article class="article" itemscope itemtype="http://schema.org/Blog" role="main">
+    <meta itemprop="interactionCount" content="UserComments:commentsCount">
```

Comment:
shouldn't this be ` <meta itemprop="interactionCount" content="UserComments:{{ commentsCount}}">`